Source/SelfService/Web/microservice/microserviceView/configuration/emptyDataTable.tsx

```diff
@@ -3,27 +3,28 @@
 
 import React from 'react';
 
-import { Paper, Typography } from '@mui/material';
+import { Paper, SxProps, Typography } from '@mui/material';
 import { AddCircle } from '@mui/icons-material';
 
 import { Button } from '@dolittle/design-system/atoms/Button';
 
-type NoConfigFilesProps = {
+type EmptyDataTableProps = {
+    title: string;
+    description: string;
+    buttonText: string;
+    sx?: SxProps
```

Comment:
Does this component need to expose `sx`?

Reply:
At the moment let it be as it is in use? Will move this to Data Grid component at some point.

<img width="992" alt="Screenshot 2022-12-12 at 16 55 57" src="https://user-images.githubusercontent.com/19160439/208140163-d750fdf6-4132-431c-91c2-a2db6b502ddc.png">